cli/gen-package/src/Elm/Gen.elm

```diff
@@ -43,17 +43,7 @@ error err =
     onFailureSend err
 
 
-{-| Report some info. The script will continue to run.
--}
-info : String -> Cmd msg
-info err =
-    onInfoSend err
-
-
 port onSuccessSend : List Json.Value -> Cmd msg
 
 
 port onFailureSend : { title : String, description : String } -> Cmd msg
-
-
-port onInfoSend : String -> Cmd msg
```

Comment:
Was `onInfoSend` not being listened to?

Reply:
Or `info`, rather?

Reply:
If `elm-review` reported it, then it's that `info` was never called in the source code.
If it isn't done already, `onInfoSend`'s subscription should be removed from JS too.

Reply:
Or, I should implement it on the JS side 😂

Reply:
Iirc it is implemented on the JS side, it's just that the `cli/gen-package` doesn't use it